src/videojs-contrib-hls.js

```diff
@@ -208,11 +208,19 @@ class HlsHandler extends Component {
         Object.defineProperty(_player, 'hls', {
           get: () => {
             videojs.log.warn('player.hls is deprecated. Use player.tech_.hls instead.');
+            // fired the uasge event when player.hls is accessed
+            tech.trigger({type: 'usage', name: 'hls-player-access'});
             return this;
           }
         });
       }
     }
+    // Object.defineProperty(tech, 'hls', {
```

Comment:
It didn't work because `tech.hls` is already a property (gets attached on `HlsHandler` creation). This event is a bit weird because we'd have to define the getter before creation, but that means an code within the project itself that accesses `tech_.hls` would fire that event. For now we can hold off on having an `hls-tech-access` event and stick with just the `hls-player-access`